app/Http/Controllers/OGImageController.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Article;
+use Illuminate\Http\Request;
+use Intervention\Image\Facades\Image;
+
+class OGImageController extends Controller {
+    private function generate($title, $author = NULL, $category = NULL){
+        $data = imagecreatefrompng('../resources/img/og_image.png');
+        $img = Image::make($data);
+
+        $lines = $this->textWrap($title, 1200, '../resources/fonts/NotoSansJP-Bold.otf', 70);
+        $center = 315;
+        $lineHeight = 100;
+        $lineY = $center - $lineHeight / 2 * (count($lines) - 1);
+        foreach ($lines as $line) {
+            $img->text($line, 600, $lineY, function ($font) {
+                $font->file('../resources/fonts/NotoSansJP-Bold.otf');
+                $font->size(70);
+                $font->align('center');
+                $font->valign('middle');
+                $font->color('#ffffff');
+            });
+            $lineY += $lineHeight;
+        }
+
+        $iconY = 500;
+        $textY = 498;
+        $pos = 145;
+
+        if ($author) {
+            $img->text('', $pos, $iconY, function ($font) {
```

Comment:
メンテナンス性に問題がありそうなので文字コード指定をしたいですね
```suggestion
            $img->text("\u{F007}", $pos, $iconY, function ($font) {
```

Reply:
(2つあるので`Add suggestion to batch`推奨)